storage/localstate_mgr.go

```diff
@@ -287,3 +294,29 @@ func getFragmentSeed(password string) ([]byte, error) {
 	seed := h.Sum(nil)
 	return seed, nil
 }
+
+func loadAddressBook(path string) ([]maddr.Multiaddr, error) {
+	path = filepath.Clean(path)
+	_, err := os.Stat(path)
+	if err != nil {
+		return nil, err
+	}
+	input, err := os.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+	data := strings.Split(string(input), "\n")
+	var peerAddresses []maddr.Multiaddr
+	for _, el := range data {
+		// we skip the empty entry
+		if len(el) == 0 {
+			continue
+		}
+		addr, err := maddr.NewMultiaddr(el)
+		if err != nil {
+			return nil, fmt.Errorf("invalid address in address book: %w", err)
```

Comment:
If for whatever reason your ` address_book` file is invalid it will just be ignored and you will only use the `address_book.seed`. The `address_book.seed` must be valid.